### PR TITLE
Added info about volunteers and echo karriere

### PR DESCRIPTION
**Pull request by heggebakk**

### CODE REVIEW

**Comment by heggebakk** [comment]
image

File: src/components/Card/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+const CardWrapper: React.FC<VolunteersInterface> = ({
+  name,
+  picture,
+  about
+}) => {
+  return (
+    <>
+      <StyledCard>
+        <StyledCardImage variant="top" src={picture} />
```
Comment (sondr3) [inline src/components/Card/Card.tsx]: Vi trenger en `alt={name}` her for accessibility.

File: src/pages/about-us/AboutUs.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      </p>
+      <h5>Lynintervju</h5>
+      <p>
+        Bedriftene har fått mulighet til å låne et rom hvor de kan holde
+        lynintervjuer av studenter de ønsker å blir bedre kjent med.
+      </p>
+      <PageTitle>Styret</PageTitle>
+      <VolunteerWrapper>
+        {volunteers.map(vol => (
+          <Card {...vol} />
```
Comment (sondr3) [inline src/pages/about-us/AboutUs.tsx]: En `key={vol.name}` kan være fint her.

**Comment by JonasTriki** [comment]
Nice PR!

**Pull request merged**